### ISSUE TITLE
[Bug] Using checkpoint-format layerwise reload in vLLM causes warning logs

### ISSUE THREAD

**Issue by andakai** [vllm-project/vime#5]
## Problem

When running Vime with the vLLM backend and native weight update, the logs contain a large number of warnings like:

```text
xxx: Failed to load weights
```

These warnings come from vLLM's layerwise reload path. In Vime's native vLLM update path, each update request sends packed partial weight buckets through IPC handles, _not a full checkpoint-style model state_. However, vLLM's `WeightTransferUpdateInfo` defaults `is_checkpoint_format` to `True`, vLLM handles these partial updates as checkpoint-format layerwise reloads.

As a result, vLLM tries to finalize/load layers that are not present in the current partial bucket and logs warnings for missing layer weights. This makes the logs noisy.

## Environment

- Branch base: `origin/main`
- vLLM version: `0.20.2`
- Megatron-Core version: `0.18.0+7371e0134`
- Model used for validation: `Qwen/Qwen3-0.6B`
- Test setup: 2 GPUs, one rollout GPU and one train GPU
- Backend: vLLM native weight update

## Reproduction

Run a short Vime vLLM backend experiment with native weight update enabled. In my local reproduction, I used Qwen3-0.6B and ran 10 training steps.

Before the change, a 10-step run produced thousands of layerwise reload warnings:

```text
Failed to load weights
```

## Expected Behavior

The update should not emit warning spam for layers that are not included in the current partial bucket.

## Suggested Fix

Set `is_checkpoint_format=False` in the native vLLM update payload in file `vime/slime/backends/vllm_utils/vllm_engine.py`:

```python
update_info = {
    "names": names,
    "dtype_names": dtype_names,
    "shapes": [list(s) for s in shapes],
    "packed": bool(packed),
    "is_checkpoint_format": False,
}
```

This makes vLLM use the non-checkpoint weight update path that is intended for direct named tensor updates.

## Validation

I tested this local change with a 10-step Qwen3-0.6B Vime run using the vLLM backend and no accuracy influence. 

If this looks right, I can open a small PR with this change.

**Comment by CalvinXKY** [comment]
We tried `is_checkpoint_format=False` on native packed `/update_weights`. **On our vLLM build it fails** (first bucket → 500, `load_weights_direct` / `param.copy_` → `leaf Variable that requires grad…`). With the default (`True`) sync works again; only the original partial-bucket warnings may remain.

**Likely a version/setup gap:** your validation was on **vLLM 0.20.2** (Qwen3-0.6B, 2 GPU); we saw this breakage on a **different vLLM revision** with Qwen3-4B / multi-engine rollout. So we’re **not merging `False` for now** until we can align versions or vLLM fixes direct load (e.g. `no_grad`).

Thanks for the report—we’ll retry after upstream or once we match 0.20.2.
